arch/risc-v/src/common/supervisor/riscv_sbi.c

```diff
@@ -125,7 +106,8 @@ void riscv_sbi_set_timer(uint64_t stime_value)
 uint64_t riscv_sbi_get_time(void)
 {
 #ifdef CONFIG_NUTTSBI
-  return sbi_mcall_get_time();
+  return sbi_ecall(SBI_EXT_FIRMWARE, SBI_EXT_FIRMWARE_GET_MTIME, 0, 0,
+                   0, 0, 0, 0);
 #else
 #ifdef CONFIG_ARCH_RV64
```

Comment:
let's merge `#else\n#ifdef` to `#elif defined`

Reply:
Thanks